Assets/Scripts/UI/InGame/HUD/FPSCounter.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using UnityEngine;
+using UnityEngine.UI;
+using UnityEngine.Rendering;
+using Newtonsoft.Json;
+public class FPSCounter : MonoBehaviour {
+
+	private float time;
+	[SerializeField] public Text counterText;
+	public Text CounterText
+	{
+		get { return counterText; }
+		set { counterText = value; }
+	}
+
+	private void Start() {
+		time = 0f;
```

Comment:
0f is the default value of an unassigned float, so no need to assign the default here